### PR TITLE
Animal death documentation

### PR DESCRIPTION
**Pull request by AndyDe-Aldent**
----

Documentation for the Animal Arrival, Animal Departure, and Animal Death resource types

----

## PR Checklist
- [x] Documentation has been checked by at least one team member
- [ ] Links work to navigate documentation
- [ ] Images stored in `_images`
- [x] Spell checked

> If some of the above are not completed why?
> no images

### CODE REVIEW

File: docs/resource-types/livestock/movements/animal-arrival.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+```
+
+| Response Item | Description | Data Type |
+| ------------- | ----------- | --------- |
+| **Id** |	The Id of this consignment | String |
+| **Scheme** |	The Scheme of this consignment | String |
+| **Origin** |	The Origin of this consignment | Location |
+| **Destination** | The Destination of this consignment | Location |
+| **Loading DateTime** | A particular point in the progression of time. Shall be UTC format with Z, specified in RFC3339 (see https://ijmacd.github.io/rfc3339-iso8601/ for format guidance). | Date/Time |
+| **Unloading DateTime** | A particular point in the progression of time. Shall be UTC format with Z, specified in RFC3339 (see https://ijmacd.github.io/rfc3339-iso8601/ for format guidance). | Date/Time |
```
Comment (VicXue) [inline docs/resource-types/livestock/movements/animal-arrival.md:120]: We should [hide the url](https://github.com/Pure-Farming/Documentation/pull/2/files#diff-e4befa26f1ced43066dbcbff27db3b698af66f4fac71dffa5839e1e2b6550e4fR73).

**Review by VicXue — CHANGES_REQUESTED** [review]

File: docs/resource-types/livestock/movements/animal-arrival.md (hunk 1 above)
Comment (AndyDe-Aldent) [reply docs/resource-types/livestock/movements/animal-arrival.md:120]: | **Loading DateTime** | A particular point in the progression of time. Shall be UTC format with Z, specified in RFC3339. | Date/Time |

File: docs/resource-types/livestock/movements/animal-death.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+| Response Item | Description | Data Type |
+| ------------- | ----------- | --------- |
+| **Country** | The country. For example, USA. You can also provide the two-letter ISO 3166-1 alpha-2 country code. | String |
+| **Locality** | The locality in which the street address is, and which is in the region. For example, Mountain View. | String |
+| **Region** | The region in which the locality is, and which is in the country. For example, California or another appropriate first-level Administrative division | String |
+| **Post Office Box** | The post office box number for PO box addresses. | String |
+| **Post Code** | The postal code. For example, 94043. | String |
+| **Street Address** | The street address. For example, 1600 Amphitheatre Pkwy. | String |
+| **Address String** | The complete address | String |
```
[the remark this answers is not in the document]
Comment (AndyDe-Aldent) [reply docs/resource-types/livestock/movements/animal-death.md]: moved all the consignment information into a new consignment.md under the movement resource type folder

File: docs/resource-types/livestock/movements/animal-arrival.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+## Response Structure
+
+A call to the Arrivals endpoints returns the following fields.
+
+```json
+{
+  "self": "string",
+  "location": {...},
+  "meta": {...},
+  "resourceType": "/livestock/movement/animal-arrivals",
```
Comment (markusPirkl) [inline docs/resource-types/livestock/movements/animal-arrival.md:60]: should be /livestock/movement/animal-arrival

File: docs/resource-types/livestock/movements/animal-arrival.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+  "id": "string"
+}
+```
+
+| Response Item | Description | Data Type |
+| ------------- | ----------- | --------- |
+|**Self** | A link to this specific Animal Arrival | URI |
+|**Location** | An identifier for the location of the Animal Arrival.| Identifier |
+|**Meta** | Meta data for the resource | Metadata |
+|**Resource Type** | The fixed discriminator for the animal arrival resource type.<br/>Value: /livestock/movement/animal-arrivals  | String |
```
Comment (markusPirkl) [inline docs/resource-types/livestock/movements/animal-arrival.md:78]: should be /livestock/movement/animal-arrival

File: docs/resource-types/livestock/movements/animal-death.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+## Response Structure
+
+A call to the Deaths endpoints returns the following fields.
+
+```json
+{
+  "self": "string",
+  "location": {...},
+  "meta": {...},
+  "resourceType": "/livestock/movement/animal-deaths",
```
Comment (markusPirkl) [inline docs/resource-types/livestock/movements/animal-death.md:60]: should be /livestock/movement/animal-death

File: docs/resource-types/livestock/movements/animal-death.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+  "explanation" : "string",
+}
+```
+
+| Response Item | Description | Data Type |
+| ------------- | ----------- | --------- |
+|**Self** | A link to this specific Animal Death | URI |
+|**Location** | An identifier for the location of the Animal Death | Identifier |
+|**Meta** | Meta data for the resource | Metadata |
+|**Resource Type** | The fixed discriminator for the animal death resource type.<br/>Value: /livestock/movement/death  | String |
```
Comment (markusPirkl) [inline docs/resource-types/livestock/movements/animal-death.md:83]: should be /livestock/movement/animal-death

File: docs/resource-types/livestock/movements/animal-death.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+# Resource Type: Animal-Death
+
+- [URLs]()
+- [Response Structure]()
+
+---
+
+## URLs
+
+Get all deaths that you have access to
```
Comment (markusPirkl) [inline docs/resource-types/livestock/movements/animal-death.md:25]: I think I would go with 

```
Get all animal death events that you have access to
```

File: docs/resource-types/livestock/movements/animal-death.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+
+```
+GET /data/livestock/movement/animal-deaths
+```
+
+Get the death for the specified id
```
Comment (markusPirkl) [inline docs/resource-types/livestock/movements/animal-death.md:31]: I think I would go with 

```
Get an individual animal death event for the specified id. 
```

File: docs/resource-types/livestock/movements/animal-death.md [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+
+```
+GET /data/livestock/movement/animal-deaths/{DeathId}
+```
+
+Get all deaths that are linked to the provided holding id
```
Comment (markusPirkl) [inline docs/resource-types/livestock/movements/animal-death.md:37]: I think I would go with 

```
Get all animal death events that you have access to for a given Holding
```

File: docs/resource-types/livestock/movements/animal-death.md [hunk 10]
```diff
@@ -0,0 +1,98 @@
...
+
+```
+GET /data/holdings/{HoldingId}/livestock/movement/animal-deaths
+```
+
+Get the death for the specified holding that is linked to the provided death id
```
Comment (markusPirkl) [inline docs/resource-types/livestock/movements/animal-death.md:43]: I think I would go with

```
Get an individual animal death event for a given Holding 
```

File: docs/resource-types/livestock/movements/animal-death.md [hunk 11]
```diff
@@ -0,0 +1,98 @@
...
+
+```
+GET /data/holdings/{HoldingId}/livestock/movement/animal-deaths/{DeathId}
+```
+
+---
+
+## Response Structure
+
+A call to the Deaths endpoints returns the following fields.
```
Comment (markusPirkl) [inline docs/resource-types/livestock/movements/animal-death.md:53]: A call to the animal death event endpoints returns the following fields.

**Comment by markusPirkl** [comment]
You need to update https://github.com/Pure-Farming/Documentation/blob/195419668a6c7d0850d1b0a8f930655922d5e5d4/docs/resource-types/index.md as well.

File: docs/resource-types/livestock/movements/animal-death.md (hunk 6 above)
Comment (markusPirkl) [reply docs/resource-types/livestock/movements/animal-death.md:83]: It looks like this is sill due to be done

**Comment by markusPirkl** [comment]
@AndyDe-Aldent as soon as this has been resolved I will approve it.  https://github.com/Pure-Farming/Documentation/blob/195419668a6c7d0850d1b0a8f930655922d5e5d4/docs/resource-types/livestock/movements/animal-death.md?plain=1#L83

Great work

**Review by markusPirkl — DISMISSED** [review]

File: docs/resource-types/livestock/movements/consignment.md [hunk 12]
```diff
@@ -0,0 +1,106 @@
...
+```
+
+| Response Item | Description | Data Type |
+| ------------- | ----------- | --------- |
+| **Id** |	The Id of this consignment | String |
+| **Scheme** |	The Scheme of this consignment | String |
+| **Origin** |	The Origin of this consignment | Location |
+| **Destination** | The Destination of this consignment | Location |
+| **Loading DateTime** | A particular point in the progression of time. Shall be UTC format with Z, specified in RFC3339. | Date/Time |
+| **Unloading DateTime** | A particular point in the progression of time. Shall be UTC format with Z, specified in RFC3339. | Date/Time |
```
Comment (VicXue) [inline docs/resource-types/livestock/movements/consignment.md:51]: I just updated the type for those two fields to `DateTimeOffset` as well to make them consistent with other date-time fields.

**Review by VicXue — APPROVED** [review]

**Pull request merged**